test/prob/test_fixture_cdf_log.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <stan/math/rev.hpp>
 #include <test/prob/utility.hpp>
+#include <test/unit/math/expect_near_rel.hpp>
```

Comment:
This should fix #1978. The changes in this file are similar to the ones in the lccdf and cdf checks.